apps/voting/test/voting.js

```diff
@@ -195,6 +218,22 @@ contract('Voting App', accounts => {
                 })
             })
 
+          it('changing required support does not affect vote required support', async () => {
+                await voting.changeSupportRequiredPct(pct16(70))
+
+                // With previous required support at 50%, vote should be approved
+                // with new quorum at 70% it shouldn't have, but since min quorum is snapshotted
+                // it will succeed
+
+                await voting.vote(voteId, true, true, { from: holder50 })
+                await voting.vote(voteId, true, true, { from: holder19 })
```

Comment:
This test is not actually testing the logic we want to test, as both holders are voting YES, the support for the proposal is 100%.

Because we are not properly using the cached `supportRequiredPct` for the `canExecute` calculation. 

We should add a NO vote by `holder31`, which would make the support of the proposal be 69, and would make this test fail until the issue with `canExecute` is fixed